content/messaging/customer-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -847,6 +862,68 @@ Apart from [Events](#events) and [Users](#users), there are also other common da
 
 </CodeResponse>
 
+## Form fields
+
+A component of the [Filled form](#filled-form) event.
+
+| Field                   | Required | Data type               |  Notes                                                                                                   |
+| ------------------------| -------- | ----------------------- | -------------------------------------------------------------------------------------------------------: |
+| `fields`                | yes      | `array of objects`      | The fields a form contains.                                                                              |
+| `fields.type`           | yes      | `string`                | Possible values: `checkbox`, `email`, `name`, `question`, `textarea`, `group_chooser`, `radio`, `select` |
+| `fields.id`             | yes      | `string`                | Field id, for all field types                                                                            |
+| `fields.label`          | yes      | `string`                | Field label; for all field types                                                                         |
+| `fields.answer`         | yes      | `any`                   | For all field types                                                                                      |
+| `fields.answer.id`      | yes      | `string`                | Answer id; for all field types                                                                           |
+| `fields.answer.label`   | yes      | `string`                | Answer label; for all field types                                                                        |
+| `fields.answer.group_id`| yes      | `number`                | For `group_chooser`                                                                                      |
```

Comment:
I think this extra `fields` indentation is not needed. After all you're describing single `field`'s content :P